tracim/tracim/lib/email_fetcher.py

```diff
@@ -146,6 +158,7 @@ def __init__(
         token: str,
         use_html_parsing: bool,
         use_txt_parsing: bool,
+        filelock_path: str,
```

Comment:
rename to `lockfile_path` (the feature is filelock because its a lock based on a file, but the filepath is the lock file path)